src/neuroconv/tools/roiextractors/roiextractors.py

```diff
@@ -272,16 +265,29 @@ def add_image_segmentation(nwbfile: NWBFile, metadata: dict) -> NWBFile:
 
 
 def add_two_photon_series(
-    imaging, nwbfile, metadata, buffer_size=10, use_times=False, two_photon_series_index: int = 0
+    imaging,
```

Comment:
Would be nice if we could add annotation types to these now that we are a it.

Reply:
Sure I'll add those after in an easy formatting/polishing PR